examples/forum/example_server.py

```diff
@@ -13,7 +13,13 @@
 
 def getCommentsFromUser(fd, usr):
     output_txt = "["
-    comment = fd.read()
+    comment = ""
```

Comment:
This works well but just FYI, IIRC I don't think you actually need this initial assignment here. Try blocks don't have their own inner scope, so any variables you create in there will be accessible outside the block. Since the except block raises an exception, the program will never pass this try block without assigning to comment.